packages/core/package.json

```diff
@@ -13,19 +12,20 @@
     "golden:test": "diff -u ./custom-elements.json ./dist/core/custom-elements.json",
     "test": "npm-run-all -s test:unit test:treeshaking",
     "test:watch": "yarn test:karma -- --auto-watch=true --single-run=false --coverage=false",
-    "test:unit": "npm-run-all -s build:ts:spec test:karma build:sass:min build:clean",
+    "test:unit": "npm-run-all -s build:ts:spec test:karma build:sass:min build:package",
     "test:karma": "karma start ./karma.conf.js",
     "test:treeshaking": "webpack --config ./test-bundles/webpack.config.js && bundlesize --config ./test-bundles/bundlesize.json",
     "test:bundles": "source-map-explorer dist/test-bundles/webpack.bundle.js",
-    "build": "npm-run-all -s build:tokens build:sass build:ts build:sass:min build:package build:clean build:api build:copy",
+    "build": "npm-run-all -s build:tokens build:sass build:ts build:sass:min build:package build:api build:copy",
     "build:watch": "npm-run-all -p build:sass:watch build:ts:watch",
-    "build:clean": "node ./package.js",
-    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '@VERSION' $npm_package_version ./dist/core/package.json; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js",
+    "build:package": "cpy '**/package.json' '../dist/core/' --cwd=./src --parents && cpy ./package.json ./dist/core/ && cpy ./README.md ./dist/core/; replace '0.0.0' $npm_package_version ./dist/core/package.json; replace '@VERSION' $npm_package_version ./dist/core/internal/utils/global.js && node ./package.js",
     "build:ts": "tsc --b ./tsconfig.project.json --force",
-    "build:ts:watch": "yarn build:ts:spec -- -w",
+    "build:ts:watch": "del ./dist/*.tsbuildinfo && tsc --p ./tsconfig.spec.json -w",
     "build:ts:spec": "tsc --b ./tsconfig.spec.json --force",
     "build:src": "npm-run-all build:sass:components build:ts",
-    "build:api": "wca analyze './dist/core/**/*.{js,ts}' --silent --format=json --outFile ./dist/core/custom-elements.json && wca analyze './dist/core/**/*.{js,ts}' --silent --format=vscode --outFile ./dist/core/vscode-html-custom-data.json",
+    "prebuild:api": "cd dist/core && yarn link",
+    "build:api": "wca analyze './dist/core/**/*.{js,ts}' --silent --format=json --outFile ./dist/core/custom-elements.json",
+    "postbuild:api": "cd dist/core && yarn unlink --no-save @clr/core",
```

Comment:
We link then unlink so the API tool can resolve the entrypoints since by default it does not respect the tsconfig path mapping.